app/tests/actions/topics-test.js

```diff
@@ -5,8 +5,9 @@ import md5 from 'spark-md5';
 import { polyfill } from 'es6-promise';
 import axios from 'axios';
 import expect from 'expect';
-import * as actions from 'actions/topics';
-import * as types from 'types';
+import sinon from 'sinon';
```

Comment:
@choonkending yeah I guess that is the logical evolution of componentalizing the project structure. This seems to be the way the winds are blowing in react land with colocation of view, data fetching, styles etc, so why not do the same with tests!